spec/query_spec.cr

```diff
@@ -126,6 +126,20 @@ describe LuckyRecord::Query do
     end
   end
 
+  describe "#raw_where" do
+    it "accepts multiple arguments and chains with #where and itself" do
+      query = UserQuery.new.where("first_name = ? AND last_name = ?", "Mikias", "Abera").where(:age, 26).query
+
+      query.statement.should eq "SELECT #{User::COLUMNS} FROM users WHERE age = $1 AND first_name = 'Mikias' AND last_name = 'Abera'"
```

Comment:
And could you also call `results` on the query like some of the other tests, just to make sure it runs successfully?